nilearn/plotting/__init__.py

```diff
@@ -34,15 +34,27 @@ def _set_mpl_backend():
 _set_mpl_backend()
 
 ###############################################################################
+import matplotlib
+from distutils.version import LooseVersion
 
 from . import cm
 from .img_plotting import plot_img, plot_anat, plot_epi, \
     plot_roi, plot_stat_map, plot_glass_brain, plot_connectome, \
     plot_prob_atlas, show
+from .surf_plotting import plot_surf, plot_surf_stat_map, plot_surf_roi
 from .find_cuts import find_xyz_cut_coords, find_cut_slices
 
 __all__ = ['cm', 'plot_img', 'plot_anat', 'plot_epi',
            'plot_roi', 'plot_stat_map', 'plot_glass_brain',
            'plot_connectome', 'plot_prob_atlas',
            'find_xyz_cut_coords', 'find_cut_slices',
            'show']
+
+# matplotlib older versions 1.1.1 will not work to plot surface data using
+# plotting functions from surf_plotting.py. Hence we check the version and
+# import them only if we have recent versions.
+
+if LooseVersion(matplotlib.__version__) > LooseVersion('1.3.1'):
+    from .surf_plotting import plot_surf, plot_surf_stat_map, plot_surf_roi
+    for plot in ['plot_surf', 'plot_surf_stat_map', 'plot_surf_roi']:
+        __all__.append(plot)
```

Comment:
It would be nicer to use the "extend" method of the list to add the full list of new names at once, rather than a for loop.